editor/src/components/editor/actions/actions.tsx

```diff
@@ -5006,6 +5010,25 @@ export const UPDATE_FNS = {
       }
     }, editor)
   },
+  SAVE_TO_GITHUB: (action: SaveToGithub, editor: EditorModel): EditorModel => {
+    const updatedRepo = parseGithubProjectString(action.targetRepository)
+    if (updatedRepo == null) {
+      return editor
+    } else {
+      const updatedEditor = {
+        ...editor,
+        githubSettings: {
+          ...editor.githubSettings,
+          targetRepository: updatedRepo,
+        },
+      }
+      // Side effect - Pushing this to the server to get that to save to Github.
+      const persistentModel = persistentModelFromEditorModel(updatedEditor)
+      saveProjectToGithub(persistentModel)
```

Comment:
Is there a plan to show the status of this in the UI yet?

Reply:
I'm currently working on a tweak that will give some feedback in the form of some toasts for success and failure. Which for the scope of this work should suffice, but once the work for loading is completed then we'll probably look at how to make more of the flow more git like.

Reply:
Yeah that makes sense 👍